serverless.yml

```diff
@@ -50,6 +62,8 @@ custom:
         sources:
           - table: cvs-local-defects
             sources: [./tests/resources/defects.json]
+          - table: cvs-local-iva-defects
+            sources: [./output_json/M1_flattened.json]
```

Comment:
Similar to the question above, can this be re-pointed to resources/iva-defects.json or is there still a need for 2 separte JSON files?